src/index.ts

```diff
@@ -71,6 +75,44 @@ const discordUserAnnouncementDictionary: { [key: string]: string } = {
   'NutDragSwag#0374': 'alexander_the_great.mp3'
 };
 
+// Event triggered when a message is sent in a text channel
+client.on('messageCreate', async (message) => {
+  // Commands are represented by a '!'
+  if (message.content.charAt(0) == '!') {
```

Comment:
Please always use `===` or `!==` due to JS inconsistency of double equals. Triple is much more reliable

Reply:
I've never known what the difference is but fixed.